src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,75 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final String PACKAGE = "mate.jdbc";
+    private static final Injector injector = Injector.getInstance(PACKAGE);
+    // i've taken out this list of main because i have a problem
+    // with declaration length usage (maven)
+    private static List<Driver> catDrivers = new ArrayList<>();
+    private static List<Driver> teslaDrivers = new ArrayList<>();
+    private static List<Driver> banderoMachineDrivers = new ArrayList<>();
+
     public static void main(String[] args) {
-        // test your code here
+        /*MANUFACTURER*/
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer caterpillar = new Manufacturer(null, "Caterpillar", "USA");
+        manufacturerService.create(caterpillar);
+        Manufacturer tesla = new Manufacturer(null, "Tesla", "USA");
+        manufacturerService.create(tesla);
+        Manufacturer banderoMachine = new Manufacturer(null, "Bandero Machine", "Ukraine");
```

Comment:
it's a bad practice to pass nulls in constructors, create new constructors to avoid such situations, fix it everywhere